File: LogicadeProgramacao/Estrutura de Decisão/2. Java/Fruteira.java
import java.util.Scanner;
public class Fruteira {
    public static void main(String[] args) {
        Scanner entrada = new Scanner(System.in);
        double qtdMorangos, qtdMacas, pesoTotal, valorMorango, valorMaca, total, desconto;
    
        System.out.print("Digite a quantidade (em Kg) de morangos: ");
        qtdMorangos = entrada.nextDouble();
        System.out.print("Digite a quantidade (em Kg) de maças: ");
        qtdMacas = entrada.nextDouble();
    
        // calcula o peso total das frutas
        pesoTotal = qtdMorangos + qtdMacas;
    
        // verifica o preço do morango de acordo com o peso
        if (qtdMorangos <= 5) {
            valorMorango = qtdMorangos * 2.5;
        } else {
            valorMorango = qtdMorangos * 2.2;
        }
    
        // verifica o preço da maçã de acordo com o peso
        if (qtdMacas <= 5) {
            valorMaca = qtdMacas * 1.8;
        } else {
            valorMaca = qtdMacas * 1.5;
        }
    
        // calcula o total da compra
        total = valorMorango + valorMaca;
    
        // verifica se o cliente tem direito a desconto
        if (pesoTotal > 8 || total > 25) {
            desconto = total * 0.1;
            total -= desconto;
        }
    
        System.out.printf("O valor total a ser pago é de: R$ %.2f", total);
        entrada.close();
    }
    
}

File: LogicadeProgramacao/Estrutura de Decisão/3. Python/Fruteira.py
qtdMorangos = float(input("Digite a quantidade (em Kg) de morangos: "))
qtdMacas = float(input("Digite a quantidade (em Kg) de maças: "))

pesoTotal = qtdMorangos + qtdMacas
quantidade_morango = float(input("Digite a quantidade (em Kg) de morangos adquirida: "))
quantidade_maca = float(input("Digite a quantidade (em Kg) de maçãs adquirida: "))

preco_morango = 0
preco_maca = 0

if quantidade_morango <= 5:
    preco_morango = 2.5
else:
    preco_morango = 2.2

if quantidade_maca <= 5:
    preco_maca = 1.8
else:
    preco_maca = 1.5

total_sem_desconto = (quantidade_morango * preco_morango) + (quantidade_maca * preco_maca)

if quantidade_morango + quantidade_maca > 8 or total_sem_desconto > 25:
    total_com_desconto = total_sem_desconto * 0.9
    print("O valor total da compra com desconto é R$ {:.2f}".format(total_com_desconto))
else:
    print("O valor total da compra é R$ {:.2f}".format(total_sem_desconto))




